protected/assets/js/script.js

```diff
@@ -135,37 +135,7 @@ $(document).ready(function(){
 			});
 		}
 	}());
-
-	(function patientWarningTooltip() {
-
-		var warning = $('.panel.patient .warning');
-		if (!warning.length) {
```

Comment:
Removing this entire block entirely seems a bit heavy handed. Would it not be better to find out why the warning is not being hidden correctly?

Reply:
I think this code block is designed to prevent the warning div from appearing if the warning doesn't exist, but could we extend it to prevent the warning div from appearing if the warning exists but has no content.

Reply:
Ok, disregard my last few comments. Instead we should add another check that returns out of this function if the '.message' span has no content

Reply:
done